config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  root to: "homepages#index"
+
+  get "homepages/index"
```

Comment:
Since you did `resoures :homepages, only: [:index]` below, you don't need this route. Also, it doesn't direct anyone anywhere! It's not quite the right syntax